storage/remote/queue_manager.go

```diff
@@ -1642,9 +1699,15 @@ func populateMinimizedTimeSeriesStr(symbolTable *rwSymbolTable, batch []timeSeri
 		case tFloatHistogram:
 			pendingData[nPending].Histograms = append(pendingData[nPending].Histograms, FloatHistogramToMinHistogramProto(d.timestamp, d.floatHistogram))
 			nPendingHistograms++
+		case tMetadata:
+			pendingData[nPending].Metadata.Type = metricTypeToMetricTypeProtoV2(d.metadata.Type)
+			// todo: intern strings
+			//pendingData[nPending].Metadata.HelpRef = d.metadata.Help
```

Comment:
What's wrong with uncommenting those?

Reply:
I need to add in the function call that does the `string -> reference #` interning :)